### DISCUSSION TITLE
Need help enabling multiple email services to receive online form submissions

### DISCUSSION THREAD

**Discussion by therealmbittarelli** [emailjs-com/help#81] [Q&A]
I see that I'm able to add more than one email service via my dashboard (and I've enabled two at this point, hoping to add one more), but so far form submissions from our site only seem to be forwarded to the email address that our EmailJS account is under. I've tried setting the second added email account as the default, thinking if only one of the two email services is going to receive the submission, it must at least be the default.. but that didn't appear to work. I've checked spam/folders. Nothing! Can anyone steer me in the right direction? Thanks!

**Comment by xr0master** [comment]
Hi. To change the recipient's address, go to your template and edit the "Email To" field.
